### PR TITLE
Issue852 cdl setpoints validation

### LINKED ISSUE #852: Add validation tests to CDL.Setpoints
The package `CDL.Setpoints` does not contain any validation tests. This issue is to add them.

### PR DESCRIPTION
**Pull request by JayHuLBL**
This closes #852. 
Validation package `CDL.SetPoints.Validation` was created.

### CODE REVIEW

File: Buildings/Experimental/OpenBuildingControl/CDL/SetPoints/Validation/HotWaterTemperatureReset.mo [hunk 1]
```diff
@@ -0,0 +1,76 @@
+within Buildings.Experimental.OpenBuildingControl.CDL.SetPoints.Validation;
+model HotWaterTemperatureReset "Test model for the heating curve"
+  extends Modelica.Icons.Example;
+  Buildings.Experimental.OpenBuildingControl.CDL.SetPoints.HotWaterTemperatureReset
```
Comment (mwetter) [inline Buildings/Experimental/OpenBuildingControl/CDL/SetPoints/Validation/HotWaterTemperatureReset.mo]: Comments are missing

File: Buildings/Experimental/OpenBuildingControl/CDL/SetPoints/Validation/TimeTable.mo [hunk 2]
```diff
@@ -0,0 +1,68 @@
+within Buildings.Experimental.OpenBuildingControl.CDL.SetPoints.Validation;
+model TimeTable "Validation model for TimeTable block"
+  extends Modelica.Icons.Example;
+  Buildings.Experimental.OpenBuildingControl.CDL.SetPoints.TimeTable
+    timTabLin(smoothness=
+        Buildings.Experimental.OpenBuildingControl.CDL.Types.Smoothness.LinearSegments, table=[0,
```
Comment (mwetter) [inline Buildings/Experimental/OpenBuildingControl/CDL/SetPoints/Validation/TimeTable.mo]: can you please reformat this so it is easier to read

**Comment by JayHuLBL** [comment]
@mwetter comments have been addressed.

**Comment by mwetter** [comment]
This is ready to merge if all tests pass.

**Pull request merged**